includes/class-sensei-settings.php

```diff
@@ -998,22 +985,6 @@ public function render_learning_mode_templates( $args ) {
 			<?php endforeach; ?>
 			</ul>
 		<?php
-
-		$inline_data = [
-			'name'         => "{$this->token}[{$key}]",
-			'value'        => $value,
-			'options'      => $args['data']['options'],
-			'customizeUrl' => Sensei_Course_Theme::get_sensei_theme_customize_url(),
-			'formId'       => "{$this->token}-form",
-			'section'      => $args['data']['section'],
-		];
-		Sensei()->assets->enqueue( 'learning-mode-templates-styles', 'course-theme/learning-mode-templates/styles.css', [ 'wp-components' ] );
-		Sensei()->assets->enqueue( 'learning-mode-templates-script', 'course-theme/learning-mode-templates/index.js', [ 'wp-element', 'wp-components' ], true );
-		wp_add_inline_script(
-			'learning-mode-templates-script',
-			sprintf( 'window.sensei = window.sensei || {}; window.sensei.learningModeTemplateSetting = %s;', wp_json_encode( $inline_data ) ),
-			'before'
-		);
```

Comment:
This is part of [LM template settings UI](https://github.com/Automattic/sensei/pull/5615). Please bring it back.

Reply:
Ah. The merge conflict and rebase kept confusing me. sorry about that.